src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -213,8 +234,22 @@ public enum MicrobeSymmetry
     /// <summary>
     ///   Current selected colour for the species.
     /// </summary>
-    [JsonProperty]
-    public Color Colour { get; set; }
+    [JsonIgnore]
+    public Color Colour
+    {
+        get => colour;
+        set
+        {
+            colour = value;
+
+            if (previewMicrobe != null && previewMicrobe.Species != null)
```

Comment:
I think these could be null coalesced together...
`previewMicrobe?.Species != null`